catalog/MDCCatalog/MDCCatalogComponentsController.swift

```diff
@@ -139,6 +145,12 @@ class MDCCatalogComponentsController: UICollectionViewController, MDCInkTouchCon
     self.view.addSubview(self.headerViewController.view)
     self.headerViewController.didMove(toParentViewController: self)
 
+    let appDelegate = UIApplication.shared.delegate as! AppDelegate
+    MDCFlexibleHeaderColorThemer.apply(appDelegate.colorScheme, toMDCFlexibleHeaderController: self.headerViewController)
+    if let backgroundColor = self.headerViewController.headerView.backgroundColor {
+      self.titleLabel.textColor = MDFTextAccessibility.textColor(onBackgroundColor: backgroundColor, targetTextAlpha: 1, options: .enhancedContrast)
```

Comment:
can we also change the statusBarStyle to match this?

Reply:
I think we should address that more generally in the issue you opened regarding the status bar style.

Reply:
Let's mention this in #1952